### ISSUE TITLE
junit report with jsx

### ISSUE THREAD

**Issue by ghost** [palcu/jest-example#3]
Hi,

I've tried your approach, but it doesn't work with jsx, althought the tests itself works perfectly without adding the jasmine-reports.

Since this repository seems like not maintained anymore, have you move to another approach about testing react components?

Thanks!

**Comment by palcu** [comment]
Yup, we've moved on a Karma+Mocha+Chai setup. Can't give you the exact approach, as we moved a little before I've quit my job though.
